src/scripting/flash/globalization/datetimenamestyle.cpp

```diff
@@ -0,0 +1,33 @@
+/**************************************************************************
+    Lightspark, a free flash player implementation
+
+    Copyright (C) 2009-2013  Alessandro Pignotti (a.pignotti@sssup.it)
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+**************************************************************************/
+
+#include "scripting/flash/globalization/datetimenamestyle.h"
+#include "scripting/class.h"
+#include "scripting/argconv.h"
+#include <iomanip>
+
+using namespace lightspark;
+
+void DateTimeNameStyle::sinit(Class_base* c)
+{
+	CLASS_SETUP_NO_CONSTRUCTOR(c, ASObject, CLASS_SEALED|CLASS_FINAL);
+	c->setVariableAtomByQName("FULL",nsNameAndKind(),asAtomHandler::fromString(c->getSystemState(),"full"),CONSTANT_TRAIT);
+	c->setVariableAtomByQName("LONG_ABBREVIATION",nsNameAndKind(),asAtomHandler::fromString(c->getSystemState(),"longAbbreviation"),CONSTANT_TRAIT);
+  c->setVariableAtomByQName("SHORT_ABBREVIATION",nsNameAndKind(),asAtomHandler::fromString(c->getSystemState(),"shortAbbreviation"),CONSTANT_TRAIT);
```

Comment:
The indentation isn't aligned to the other lines.

Reply:
Should be fine now. Need to tweek my text editor a little so I can see this next time.